### PR TITLE
How we calculate (part 2)

### PR DESCRIPTION
**Pull request by OlaShabalina**
## Overview of the Pull Request:

Hey team, 

### This PR includes:
* add text blocks for all sections with content;
* provide links to referenced resources;
* add anchor tags for on-page navigation using the table of content;
* add a link back to the main page;
* run Prettier;

### Out of scope for this PR:
* convert CalculationFormula into text block with cvg`s (next PR);
* calculationFormula is messing up responsiveness atm, will be fixed once converted to text, so please disregard;
* button to go back up to the beginning of the page';
* proof read the text (used the one checked by Sean but just in case :-) 

## How Has This Been Tested?
- Load to your machine;
- Check all links in the table of content;
- Check the link to go back to the results page - now both pages are connected;
- Check the links for all referenced resources:
  - Code for Australia;
  - Civic Makers;
  - GHGProtocol

## Pull Request Checklist:

Make sure the following checkboxes`[ ]` are checked with `x` before sending your PR -- thank you!

~- [ ] Have you included a link Trello card / Github issue and written sufficient description to help others review your work?~
- [x] Is your pull request up-to-date with `main` branch?
- [x] Have you checked That code is well formatted? Did `npx prettier --check .` return no warnings/errors?
- [x] Have you written instructions on how to test that your changes work as expected?
- [x] Have you tested your work thoroughly on your local machine to ensure you are not breaking anything?

### CODE REVIEW

File: pages/howWeCalculate.js [hunk 1]
```diff
@@ -1,41 +1,135 @@
...
+              color="#044B7F"
+              textDecoration="underline"
+              textUnderlineOffset="3px"
+              fontWeight={700}
+              href="https://www.codeforaustralia.org/"
+            >
+              Code for Australia
+            </Link>
+            , a not-for-profit organisation that builds tech stuff that matters
+            for all levels of government.
```
Comment (hqtan) [inline pages/howWeCalculate.js:114]:
```suggestion
            , an organisation that builds tech that matters for all
            levels of government.
```

File: pages/howWeCalculate.js [hunk 2]
```diff
@@ -1,41 +1,135 @@
...
       >
         <Flex width={["100%", "70%"]} direction="column">
+          <Link
+            fontWeight={700}
+            fontSize="16px"
+            pb={["20px", "30px"]}
+            href="/results"
+          >
+            Work Commute Survey Results / Carbon Emissions Calculating and
+            reporting
```
Comment (hqtan) [inline pages/howWeCalculate.js:23]: I think we can make the Back link title a little more terse:
```suggestion
            Back to Work Commute Survey Results
```
and maybe add the 'Back' Arrow icon (that we use in the survey forms) along with title, to make it more apparent that this is a clickable title?

image

**Review by hqtan — COMMENTED** [review]
Most Excellent work on the HWC page @OlaShabalina ! 🙌🏽 

I left a couple of suggestions to ponder on, but they need not be acted upon at this point (we can revisit them in a later PR) 👍🏽

**Review by hqtan — APPROVED** [review]

File: pages/howWeCalculate.js (hunk 2 above)
Comment (OlaShabalina) [reply pages/howWeCalculate.js:23]: We can bring it up to design team on Tuesday if you like, I implemented the current figma design. 👍

**Pull request merged**